olp-cpp-sdk-authentication/src/AuthenticationClient.cpp

```diff
@@ -925,6 +1002,11 @@ client::CancellationToken AuthenticationClient::Impl::IntrospectApp(
       return client::ApiError({http_result.status, msg});
     }
 
+    if (document.HasParseError()) {
+      return client::ApiError({static_cast<int>(http::ErrorCode::UNKNOWN_ERROR),
+                               "Failed to parse response."});
```

Comment:
```suggestion
                               "Failed to parse response"});
```
Either you have everywhere `.` or you don't. Please keep consistent.